style.css

```diff
@@ -313,28 +309,37 @@ body {
   margin-top: 50px;
 }
 
-.skill-list-container .skill-section {
-  height: 24px;
+.skill-section {
   line-height: 24px;
   color: black;
   display: flex;
+  flex-direction: column;
   justify-content: space-between;
-  align-items: center;
+  margin: 0;
 }
```

Comment:
- [ ] Please, make sure to have proper padding/margin for the skill section.

Your design:
![image](https://user-images.githubusercontent.com/86069740/236355940-fe20333a-636f-40a1-aefd-6958bde962cc.png)

Figma:
![image](https://user-images.githubusercontent.com/86069740/236355984-5b484a99-bc01-4260-ac04-c19fad5087f3.png)

- [ ] Kindly, also remove the extra line above the Frameworks section title.
Your design:
![image](https://user-images.githubusercontent.com/86069740/236356302-94fb40e1-600b-4909-ac5f-993ec396011e.png)

Figma design:
![image](https://user-images.githubusercontent.com/86069740/236356332-69e99e7a-61d6-45c7-8837-9a25efae796c.png)